### PR TITLE
Refactoring topics from topics.js to YAML topics file in common

### PR DESCRIPTION
**Pull request by ericyem**
## Description

Refactored the topics.js file to use the YAML topics file in the common repository. Also allowed display of the camera IP address.

## Screenshots

image

## Steps to Test

1. Run the dasboard
2. Run `poetry shell` and then `python orchestrator.py` in the raspicam repository
3. Should display the camera system to be online, with the ip address
4. Other tests include running `mosquitto_pub -t "/v3/wireless_module/2/status" -h localhost -m "{\"online\": true}"` and then running `python -m das.V3_fake_module -t 10 -r 3` from the data-acquisition repository. Should be working the same without the topics file.

### CODE REVIEW

File: server/js/sockets.js [hunk 1]
```diff
@@ -168,8 +167,8 @@ sockets.init = function socketInit(server) {
           const value = JSON.parse(payloadString);
           const path = topicString.slice(1); // Path is from "wireless_module"
           // Module's online
+          console.log(retained);
```
Comment (kdav108) [inline server/js/sockets.js]: Nit but we can possible remove this if not needed.

File: server/js/sockets.js [hunk 2]
```diff
@@ -168,8 +167,8 @@ sockets.init = function socketInit(server) {
...
           if (property === 'start') {
-            retained[path[0]].online = true;
```
Comment (kdav108) [inline server/js/sockets.js:172]: It's been so long, do we remember why we don't need this anymore?

**Review by kdav108 — APPROVED** [review]
Looks good to me! Couldn't test with raspicam but the interaction with WM is smooth.

File: client/src/components/v3/status/CameraStatusContainer.tsx [hunk 3]
```diff
@@ -21,15 +21,15 @@ export default function CameraStatusContainer(): JSX.Element {
...
+    ip: useCameraStatus('primary')?.ipAddress ?? 'Error no IP',
     battery: useCameraBattery('primary')?.voltage ?? null,
     videoFeedEnabled: useVideoFeedStatus('primary')?.online ?? null,
   };
 
   const secondaryProps: CameraStatusProps = {
     cameraName: getPrettyDeviceName('secondary'),
     online: useCameraStatus('secondary')?.connected ?? false,
-    ip: '501 not implemented', // TODO: Use API
+    ip: useCameraStatus('primary')?.ipAddress ?? 'Error no IP', // TODO: Use API
```
Comment (hallgchris) [inline client/src/components/v3/status/CameraStatusContainer.tsx]: This TODO has been addressed!

**Review by hallgchris — APPROVED** [review]
This will break CICD as yarn doesn't have credentials to get the mhp repo. I can look into this sometime, don't worry about it here.

I also noticed that the camera only appears online if it becomes online while the page is open. Navigating away and back again breaks it. Again not caused by this PR, but does need fixing.

Enough of that though, this PR looks good! Just some minor things to tidy up.

**Pull request merged**